rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -101,20 +100,16 @@
     private Keccak256 hash;
     private Keccak256 rawHash;
 
-    protected Transaction(byte[] rawData) {
-        RLPList transaction = RLP.decodeList(rawData);
-
+    protected Transaction(RLPList transaction) {
```

Comment:
Why did you change the byte array for RLPList? whats the rationale?
And why didn't you change it in subclasses?

Reply:
@patogallaiovlabs I've tried to reduce amount of `byte[]` arrays passed around as arguments, which brings a stronger typing. 
It might be unnecessary here, as subclasses seem to be about of scope of this task.